Framework/DataHandling/src/SaveAscii2.cpp

```diff
@@ -275,63 +275,14 @@ void SaveAscii2::exec() {
   file.close();
 }
 
-/**writes a spectra to the file using an iterator
-@param spectraItr :: a set<int> iterator pointing to a set of workspace IDs to
-be saved
+/** Writes a spectrum to the file using a workspace index
+@param wsIndex :: an integer relating to a workspace index
 @param file :: the file writer object
 */
-void SaveAscii2::writeSpectra(const std::set<int>::const_iterator &spectraItr,
```

Comment:
Where there simply two identical versions of the same method, one for an index, one for an iterator to an index?